### PR TITLE
Add dashboards for v2 of mssql metrics

### PR DESCRIPTION
**Pull request by StefanKurek**

### CODE REVIEW

File: dashboards/sqlserver/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#### Notes
+
+- This dashboard is based on Google's [Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent).
+- The full list of metrics supported by the Ops Agent can be found [here](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mssql#monitored-metrics).
+
+|SQL Server GCE Overview|
+|:------------------|
+|Filename: [sqlserver-gce-overview.json](sqlserver-gce-overview.json)|
+|This dashboard has charts for viewing SQL Server when monitored by 
+Google's Ops Agent including `User Connections`, `Avg Lock Wait Time`, `Lock Waits`, `Batch Info`, `Buffer Cache Hit Ratio`, `Page Checkpoint Flushes`, `Lazy Writes`, `Page Life Expectancy`, `Page Splits`, `Page Operations`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`.
```
Comment (EvanSimpson) [inline dashboards/sqlserver/README.md:12]:
```suggestion
Google's Ops Agent including `User Connections`, `Avg Lock Wait Time`, `Lock Waits`, `Batch Info`, `Buffer Cache Hit Ratio`, `Page Checkpoint Flushes`, `Lazy Writes`, `Page Life Expectancy`, `Page Splits`, `Page Operations`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs`, and `Hosts by Region`.|
```

File: dashboards/sqlserver/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+|SQL Server Transaction Logs|
+|:------------------|
+|Filename: [sqlserver-transaction-logs.json](sqlserver-transaction-logs.json)|
+|This dashboard has charts for viewing SQL Server Transaction Log data when monitored by 
+Google's Ops Agent including `Log Flush Waits`, `Log Flushes`, `Log Flush Data`, `Log Usage`, `Log Growths`, and `Log Shrinks`.
```
Comment (EvanSimpson) [inline dashboards/sqlserver/README.md:18]:
```suggestion
|This dashboard has charts for viewing SQL Server Transaction Log data when monitored by Google's Ops Agent including `Log Flush Waits`, `Log Flushes`, `Log Flush Data`, `Log Usage`, `Log Growths`, and `Log Shrinks`.|
```

File: dashboards/sqlserver/sqlserver-transaction-logs.json [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+            }
+          }
+        },
+        "width": 6,
+        "xPos": 6,
+        "yPos": 8
+      }
+    ]
+  },
+  "name": "projects/1086931010737/dashboards/cc540920-92cf-4299-a166-03c8cfb36adc"
```
Comment (EvanSimpson) [inline dashboards/sqlserver/sqlserver-transaction-logs.json:248]:
```suggestion
  }
```

File: dashboards/sqlserver/sqlserver-gce-overview.json [hunk 4]
```diff
@@ -0,0 +1,522 @@
+{
+  "category": "CUSTOM",
+  "displayName": "SQL Server GCE Overview",
+  "etag": "3beb3af8ee3eab21b63f026ed5f5d810",
+  "mosaicLayout": {
```
Comment (EvanSimpson) [inline dashboards/sqlserver/sqlserver-gce-overview.json:5]:
```suggestion
  "displayName": "SQL Server GCE Overview",
  "mosaicLayout": {
```

File: dashboards/sqlserver/sqlserver-gce-overview.json [hunk 5]
```diff
@@ -0,0 +1,522 @@
...
+          },
+          "title": "SQL Server Monitoring Links"
+        },
+        "width": 6,
+        "xPos": 0,
+        "yPos": 20
+      }
+    ]
+  },
+  "name": "projects/1086931010737/dashboards/119ffa28-8f19-4f32-acb5-2e68bcea8209"
```
Comment (EvanSimpson) [inline dashboards/sqlserver/sqlserver-gce-overview.json:521]:
```suggestion
  }
```

File: dashboards/sqlserver/sqlserver-transaction-logs.json [hunk 6]
```diff
@@ -0,0 +1,249 @@
+{
+  "category": "CUSTOM",
+  "displayName": "SQL Server Transaction Logs",
+  "etag": "d8c1e07cf7700a75f483f5182126bef9",
+  "mosaicLayout": {
```
Comment (EvanSimpson) [inline dashboards/sqlserver/sqlserver-transaction-logs.json:5]:
```suggestion
  "displayName": "SQL Server Transaction Logs",
  "mosaicLayout": {
```

**Review by EvanSimpson — CHANGES_REQUESTED** [review]
Thanks for the PR! I suggested a few edits to fix formatting and remove project-specific fields from the dashboard json.

**Comment by schmikei** [comment]
@EvanSimpson I think everything should be resolved! Along with this being merged https://github.com/GoogleCloudPlatform/ops-agent/pull/554, I think this should be good to go now as well.

**Comment by EvanSimpson** [comment]
@schmikei thanks for the update! Can you add a metadata.yaml file as well - that's been introduced since this PR was opened back in April. You can see an example here https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/iis/metadata.yaml, and there's some documentation in the repo-level readme.

I'll merge this once the Ops Agent release with the V2 metrics has gone out!

**Comment by schmikei** [comment]
Sounds good I'll get right on it! Thanks @EvanSimpson!

File: dashboards/sqlserver/metadata.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+- category: Microsoft SQL Server
+  id: sqlserver-gce-overview
+  display_name: SQL Server GCE Overview
+  description: "This dashboard has charts for viewing MSSQL when monitored by [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mssql) including User Connections, Avg Lock Wait Time, Lock Waits, Batch Info, Buffer Cache Hit Ratio, Page Checkpoint Flushes, Lazy Writes, Page Life Expectancy, Page Splits, Page Operations, CPU % Top 5 VMs, Memory % Top 5 VMs, and Hosts by Region"
+  integration_platform: GCE
```
Comment (EvanSimpson) [inline dashboards/sqlserver/metadata.yaml:5]: Can you update the format of the integration fields here? Should be `related_integrations` list. You can find documentation here https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples#metadatayaml-content

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/sqlserver/metadata.yaml (hunk 7 above)
Comment (schmikei) [reply dashboards/sqlserver/metadata.yaml]: I think should be resolved now, mind taking a second look?

File: dashboards/sqlserver/metadata.yaml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+- category: Microsoft SQL Server
+  id: sqlserver-gce-overview
+  display_name: SQL Server GCE Overview
+  description: "This dashboard has charts for viewing MSSQL when monitored by [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mssql) including User Connections, Avg Lock Wait Time, Lock Waits, Batch Info, Buffer Cache Hit Ratio, Page Checkpoint Flushes, Lazy Writes, Page Life Expectancy, Page Splits, Page Operations, CPU % Top 5 VMs, Memory % Top 5 VMs, and Hosts by Region"
+  related_integrations:
+    - id: mssql
+      platform: mssql
```
Comment (EvanSimpson) [inline dashboards/sqlserver/metadata.yaml:7]:
```suggestion
      platform: GCE
```

File: dashboards/sqlserver/metadata.yaml [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+      version: "2"
+
+- category: Microsoft SQL Server
+  id: sqlserver-gce-transaction-logs
+  display_name: SQL Server Transaction Logs
+  description: "This dashboard has charts for viewing SQL Server Transaction Log data when monitored by Google's Ops Agent including Log Flush Waits, Log Flushes, Log Flush Data, Log Usage, Log Growths, and Log Shrinks"
+  related_integrations:
+    - id: mssql
+      platform: mssql
```
Comment (EvanSimpson) [inline dashboards/sqlserver/metadata.yaml:16]:
```suggestion
      platform: GCE
```

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/sqlserver/metadata.yaml (hunk 7 above)
Comment (EvanSimpson) [reply dashboards/sqlserver/metadata.yaml]: Suggested fix for `platform: GCE`, then LGTM

File: dashboards/sqlserver/metadata.yaml (hunk 9 above)
Comment (schmikei) [reply dashboards/sqlserver/metadata.yaml]: Ope sorry about that!

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**